dotcom-rendering/src/components/YoutubeAtom/YoutubeAtom.tsx

```diff
@@ -178,6 +140,15 @@ export const YoutubeAtom = ({
 	 */
 	const playerReadyCallback = useCallback(() => setPlayerReady(true), []);
 
+	const YoutubeAtomOverlayWithCommonProps = cloneElement(YoutubeAtomOverlay, {
+		uniqueId,
+		posterImage,
+		title,
+		height,
+		width,
+		onClick: () => setOverlayClicked(true),
+	});
```

Comment:
The use of `cloneElement` is discouraged:
https://react.dev/reference/react/cloneElement

The docs suggest using a render prop instead but I'm not sure why we create an overlay component and pass that in as a prop and then clone it later with more props. Duplicating the props further up may help.

However I'm wondering another approach might be to push down the decision of which overlay to use and delegated to a function like `decideOverlay`